src/jdk.jcmd/share/man/jcmd.1

```diff
@@ -402,12 +378,6 @@ On Linux operating systems, the temporary directory is \f[V]/tmp\f[R].
 On Windwows, the temporary directory is specified by the \f[V]TMP\f[R]
 environment variable.)
 .IP \[bu] 2
-\f[V]preserve-repository=\f[R]{\f[V]true\f[R]|\f[V]false\f[R]} :
-Specifies whether files stored in the disk repository should be kept
-after the JVM has exited.
-If false, files are deleted.
-By default, this parameter is disabled.
-.IP \[bu] 2
```

Comment:
Why is this being removed?

Reply:
I'm guessing the src used to generate the manpage was missing the very recent change.

Reply:
Yes, it was rush hour in the jcmd man page, the changes overlapped.  Updated this change.